### DISCUSSION TITLE
安装o3de的时候遇到了0x80070643这样的错误

### DISCUSSION THREAD

**Discussion by Zaran-12** [o3de/o3de#17564] [Q&A]
我装了cmake
![image](https://github.com/o3de/o3de/assets/135086343/031f137d-b54d-4059-b53e-768bd6e9faa8)
git
![image](https://github.com/o3de/o3de/assets/135086343/0ceb3854-fc16-4159-abaa-66bd2335450f)
visual studio2022 并增加了他应该需要的工作桌面负荷，
可是为什么会出现这样的状况，，哪位大佬可以帮帮我

![屏幕截图 2024-03-03 231848](https://github.com/o3de/o3de/assets/135086343/e1b51514-fb8d-4d3a-9975-9226b129274a)
附件有我的日志
[O3DE_20240303213826.log](https://github.com/o3de/o3de/files/14473510/O3DE_20240303213826.log)

**Comment by kiemz** [comment]
hello 你找到解决办法了吗

**Reply by kiemz** [reply]
不知道你解决没有，可以尝试下面的内容
1.不要安在c盘，可能有权限问题
2.执行下列命令
Dism /Online /Cleanup-Image /CheckHealth  是检查映像以查看是否有检测到损坏
Dism /Online /Cleanup-Image /ScanHealth   是扫描你全部系统文件并和官方系统文件对比
Dism /Online /Cleanup-Image /RestoreHealth   是把那些不同的系统文件还原成系统官方源文件，跟重装差不多
sfc /scannow  修复还原

注意:
如果第3行命令卡在62.3，可以先执行第4行，确保命令都执行完

参考链接: https://answers.microsoft.com/zh-hans/windows/forum/all/%E6%B1%82%E5%8A%A9%E5%85%B3%E4%BA%8E%E5%AE%89/448b7af3-a3aa-423f-880d-9b262f09e835
